commands/config_e2e_test.go

```diff
@@ -1,14 +1,56 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
+
+// +build e2e
+
 package commands
 
 import (
+	"github.com/mattermost/mattermost-server/v5/model"
 	"github.com/spf13/cobra"
 
 	"github.com/mattermost/mmctl/client"
 	"github.com/mattermost/mmctl/printer"
 )
 
+func (s *MmctlE2ETestSuite) TestConfigResetCmdE2E() {
+	s.SetupTestHelper().InitBasic()
+
+	_, appErr := s.th.App.CreateUser(&model.User{Email: s.th.GenerateTestEmail(), Username: model.NewId(), Password: model.NewId()})
+	s.Require().Nil(appErr)
```

Comment:
Why are we creating a new user here? I dont think it is required.

Reply:
will remove this